docs/reference/setup/install/rpm.asciidoc

```diff
@@ -123,6 +123,17 @@ endif::[]
 
 include::skip-set-kernel-parameters.asciidoc[]
 
+[[install-rhel-security]]
+==== Override Red Hat TMP if Security Compliance
+
+If you are applying a security compliance in your RHEL installation 
+you must change the path of the TMP directory that will use elasticsearch as Java.
+
+1. Uncomment `-Djava.io.tmpdir=${ES_TMPDIR}` under `/etc/elasticsearch/jvm.options`
```

Comment:
I think we shouldn't be telling users to edit `jvm.options`, they should be adding extra options files in `jvm.options.d`.

But then I think this isn't necessary anyway, we already copy `ES_TMPDIR` to `java.io.tmpdir` IIRC.